src/test/ebpf_yaml.cpp

```
@@ -489,59 +488,57 @@ void add_stack_variable(std::set<std::string>& more, int& offset, const std::vec
-    return stack_contents_invariant(bytes);
+    return stack_contents_invariant(bytes, total_stack_size);
 }
 
 ConformanceTestResult run_conformance_test_case(const std::vector<uint8_t>& memory_bytes,
                                                 std::span<const EbpfInst> instructions, bool debug) {
-    thread_local_options = {};
+    ebpf_verifier_options_t options{};
     ebpf_context_descriptor_t context_descriptor{64, -1, -1, -1};
     EbpfProgramType program_type = make_program_type("conformance_check", &context_descriptor);
 
     ProgramInfo info{&g_platform_test, {}, program_type};
```

Cppcheck (2.20.0)

[style] The function 'close_after_meet' is never used.

(unusedFunction)